src/json.cpp

```diff
@@ -357,9 +357,8 @@ static void to_surrogate_pair(uint32_t unicode, uint16_t *uc, uint16_t *lc)
   *lc = (n & 0x3FF) | 0xDC00;
 }
 
-#define is_space(c) ((c) == '\t' || (c) == '\n' || (c) == '\r' || (c) == ' ')
-#define is_digit(c) ((c) >= '0' && (c) <= '9')
-
+bool is_space        (const char *c);
+bool is_digit        (const char *c);
```

Comment:
In what way? I declared it as static bool and some of the windows and os x
builds failed. This declaration passed all builds though.
On Sun, Jul 26, 2015 at 10:16 PM Marcel Greter <notifications@github.com>
wrote:

> In src/json.cpp
> <https://github.com/sass/libsass/pull/1373#discussion_r35503582>:
>
> > @@ -357,9 +357,8 @@ static void to_surrogate_pair(uint32_t unicode, uint16_t *uc, uint16_t *lc)
> >    *lc = (n & 0x3FF) | 0xDC00;
> >  }
> >
> > -#define is_space(c) ((c) == '\t' || (c) == '\n' || (c) == '\r' || (c) == ' ')
> > -#define is_digit(c) ((c) >= '0' && (c) <= '9')
> > -
> > +bool is_space        (const char *c);
> > +bool is_digit        (const char *c);
>
> This declaration doesn't seem to match the implementation?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sass/libsass/pull/1373/files#r35503582>.
>